tests/testthat/test-fatal-errors.R

```diff
@@ -1,12 +1,10 @@
 test_that("Unbalanced braces", {
```

Comment:
Now we can test fatal errors safely, R does not crash any more

Reply:
Huzzah!